version 5

```diff
@@ -205,11 +159,7 @@
 						"type": "url"
 					}
 				},
-				{
-					"render": {
-						"en": "{reviews(name)}"
-					}
-				},
+				"reviews",
```

Comment:
Again: "reviews" is a builtin tagrendering. For reference, [these are defined here](https://github.com/pietervdvn/MapComplete/blob/develop/assets/tagRenderings/questions.json)